pkgs/development/python-modules/mailsuite/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, lib, buildPythonPackage, fetchPypi, isPy3k, pythonPackages }:
+
+buildPythonPackage rec {
+  pname = "mailsuite";
+  version = "1.6.1";
+
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "17bsnfjjzv8hx5h397p5pa92l6cqc53i0zjjz2p7bjj3xqzhs45a";
+  };
+
+  propagatedBuildInputs = with pythonPackages; [
```

```suggestion
  propagatedBuildInputs = [
```

please list them as individual inputs.

---

Fixed!